.changeset/pre.json

```diff
@@ -1,5 +1,5 @@
 {
-  "mode": "pre",
+  "mode": "exit",
```

Comment:
This should be `pre` still

Reply:
For some reason the publish keeps failing without this: https://github.com/xataio/client-ts/actions/runs/7613023749/job/20731937567